lib/Twig/Extension/Core.php

```diff
@@ -852,6 +852,34 @@ function twig_in_filter($value, $compare)
     return false;
 }
 
+/** 
+ * Returns a trimmed string.
+ * 
+ * @param string $string
+ * @param string $characterMask
+ * @param string $mode
+ * @return string
+ */
+function twig_trim_filter($string, $characterMask = null, $mode = 'both')
+{
+    if (empty($characterMask)) {
+        $characterMask = " \t\n\r\0\x0B";
```

Comment:
An empty string should not make the function function work like passing no parameter/passing null.

And this would be a BC break since currently `|trim('')` doesn't trim anything.